hathor/transaction/util.py

```diff
@@ -59,6 +59,10 @@ def get_withdraw_amount(settings: HathorSettings, melt_amount: int) -> int:
     return floor(abs(settings.TOKEN_DEPOSIT_PERCENTAGE * melt_amount))
 
 
+def get_token_amount_from_htr(settings: HathorSettings, htr_amount: int) -> int:
```

Comment:
Missing docstring. I didn't understand what this function is doing by reading the code.

Reply:
Thanks!